sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -1050,55 +1053,73 @@ public String getDefaultLanguage() {
     }
 
     /**
-     * Cancel a long-running operation healthcare task by given a healthcare task identification number.
+     * Execute actions, such as, entities recognition, PII entities recognition and key phrases extraction in a list of
+     * {@link String document} with provided request options.
```

Comment:
for a list of {@link document**s**}

Reply:
we are using a list of document in all other endpoints.